app/models/agents/read_file_agent.rb

```diff
@@ -0,0 +1,49 @@
+module Agents
+  class ReadFileAgent < Agent
+    include FormConfigurable
+    include FileHandling
+
+    cannot_be_scheduled!
+
+    def default_options
+      {
+        'data_key' => 'data'
+      }
+    end
+
+    description do
+      <<-MD
+        The ReadFileAgent takes events from `FileHandling` agents, reads the file, and emits the contents as a string.
```

Comment:
Before we merge, it might be nice to add a wiki page about FileHandling agents and link to it in each of them.